lib/Doctrine/Inflector/Rules/Dutch/Inflectible.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Rules\Pattern;
+use Doctrine\Inflector\Rules\Substitution;
+use Doctrine\Inflector\Rules\Transformation;
+use Doctrine\Inflector\Rules\Word;
+
+/**
+ *  // http://nl.wikipedia.org/wiki/Klinker_(klank)
+ *  private $klinker = '(a|e|i|o|u|ij)';
+ *  private $korteKlinker = '(u|i|e|a|o)';          // @todo '(ie|oe)'. $plofKlank
+ *
+ *  // http://nl.wikipedia.org/wiki/Medeklinker
+ *  private $plofKlank = '(p|t|k|b|d)';
+ *  private $wrijfKlank = '(f|s|ch|sj|v|z|g|j)';    // journaal
+ *  private $neusKlank = '(m|n|ng)';
+ *  private $vloeiKlank = '(l|r)';
+ *  private $glijKlank = '(j|w)';                   // jaar
+ *  private $affricate = '(ts|zz|tsj|g)';           // /d3/ gin
+ *  private $missingFromWiki = '(c|h|p|q|x|y|z)';
+ *
+ *  public function __construct()
+ *  {
+ *      $this->medeklinker = '(' . $this->missingFromWiki . '|' . $this->plofKlank . '|' . $this->wrijfKlank . '|' . $this->neusKlank . '|' . $this->vloeiKlank . '|' . $this->glijKlank . '|' . $this->affricate . ')';
+ *      $this->medeklinker = '((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))';
+ *  }
+ */
+class Inflectible
+{
+    /**
+     * @return Transformation[]
+     */
+    public static function getSingular() : iterable
+    {
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Klinkerverandering
+        yield new Transformation(new Pattern('()heden$'), '\1heid');
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Beroepen_eindigend_op_-man
+        yield new Transformation(new Pattern('()mannen$'), '\1man');
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Latijnse_meervoudsvormen
+        yield new Transformation(new Pattern('()ices$'), '\1ex');
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Stapelmeervoud
+        yield new Transformation(new Pattern('^(ei|gemoed|goed|hoen|kind|lied|rad|rund)eren$'), '\1');
+
+        // http://nl.wikipedia.org/wiki/Nederlandse_grammatica
+        yield new Transformation(new Pattern('()ijen$'), '\1ij');
+
+        yield new Transformation(new Pattern('()ieen$'), '\1ie');       // ën
+
+        yield new Transformation(new Pattern('()((a|e|i|o|u|ij))s$'), '\1\2');
+
+        yield new Transformation(new Pattern('()((s)s)en$'), '\1s');
+
+        yield new Transformation(new Pattern('()((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))en$'), '\1\2');
+    }
+
+    /**
+     * @return Transformation[]
+     */
+    public static function getPlural() : iterable
+    {
+        // allready in plural
```

Comment:
It's spelt "already", and I don't understand what this comment is about…

Reply:
corrected.

Reply:
Let's remove it then?

Reply:
may it be let in, as a remider?

Reply:
a reminder of what? We have no idea what it means…